nilearn/input_data/nifti_spheres_masker.py

```diff
@@ -115,6 +173,33 @@ def _iter_signals_from_spheres(seeds, niimg, radius, allow_overlap,
         yield X[:, row]
 
 
+def _iter_regions_from_spheres(seeds, target_affine, target_shape, radius, 
```

Comment:
providing mask_img as an alternative to `target_affine` and `target_shape` would be better I think